backend/node/base.py

```diff
@@ -266,10 +276,7 @@ async def _run_genvm(
             self.contract_snapshot, self.contract_snapshot_factory, readonly
         )
 
-        # FIXME(core-team): please provide valid date
-        transaction_datetime = datetime.datetime.fromisoformat(
-            "2024-11-28T06:42:42.424242Z"
-        )
+        transaction_datetime = datetime.datetime.fromisoformat(transaction_created_at)
```

Comment:
```py
>>> datetime.datetime.fromisoformat(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: fromisoformat: argument must be str
```

Reply:
This one is weird, probably not appropriately updated because that's not the latest version.
For the None case, we can set a default, but that should be managed when the transaction is created.